realm/realm-annotations-processor/src/test/resources/io/realm/BooleansRealmProxy.java

```diff
@@ -15,7 +13,9 @@
 import io.realm.internal.Row;
 import io.realm.internal.SharedRealm;
 import io.realm.internal.Table;
+import io.realm.internal.UncheckedRow;
```

Comment:
This looks like a mistake that might trigger unused import warnings by static analysis tools.